config/locales/en.yml

```diff
@@ -80,6 +82,14 @@ en:
     disable: Disable
     upload: Upload your certificate
     previous: Previous
+    errors:
+      invalid: is not a valid x509 certificate
+      expired: has expired
+      expires_soon: expires too soon
+      valid_too_long: valid for too long
+      not_rsa: is not RSA
+      small_key: key size is less than 2048 bits
+      not_signing: signing certificate required
```

Comment:
Should this be `no_signing`?

Reply:
The context is someone is trying to disable an encryption cert, though it's only possible to disable signing ones. So saying it's not signing...

Reply:
👍